src/home/rendering/display-popup-modal.ts

```diff
@@ -20,3 +20,12 @@ export function displayPopupMessage(header: string, message: string, url?: strin
     gitHubLoginButton?.classList.add("highlight");
   }
 }
+
+export function showError(error: string, showToast = false, description?: string) {
+  console.error(description, error);
```

Comment:
Shouldn't this be `error` first and then description since the `description` is the one which is optional?